MAINTAINERS.md

```diff
@@ -15,6 +15,4 @@
 | Ryan Beck-Buysse | rbuysse |
 | Shannyn Telander | shannynalayna |
 | Shawn Amundson | vaporos |
-| Jason Walker | desktophero |
```

Comment:
This removal should be in the first commit, not this one.

Reply:
Fixed.